python/ray/tune/examples/genetic_example.py

```diff
@@ -0,0 +1,61 @@
+"""This test checks that GeneticSearch is functional.
```

Comment:
Let's make sure this is run (i.e., on Jenkins in `test/jenkins/multi_node_tests`)

Reply:
Did you mean this directory: `test/jenkins_tests/multi_node_tests/`? I see some ray tests there, but there is no ray.tune tests. Is there a place to run all the examples? Or maybe I could add a run_automl.sh to wrap this example and run

Reply:
Yeah - this example should be run in this file around here: https://github.com/ray-project/ray/blob/master/test/jenkins_tests/run_multi_node_tests.sh#L276